From: SeongJae Park <sj@kernel.org>
Date: Mon, 8 Sep 2025 19:08:20 -0700
Subject: [PATCH] ==== samples init enable fixup fixup ====

samples/damon: fix boot time enable handling fixup merge mistakes

First three patches of the patch series "mm/damon: fix misc bugs in
DAMON modules" [1] was trying to fix boot time DAMON sample modules
enabling issues by avoiding starting DAMON before the module
initialization phase.  However, probably by a mistake during a merge,
only half of the change is merged, and the part for avoiding the
starting of DAMON before the module initialized is missed.  So the
problem is not solved.  Fix those.

Note that the broken commits are merged into 6.17-rc1, but also
backported to relevant stable kernels.  So this series also need to be
merged into the stable kernels.  Hence Cc-ing stable@.

[1] https://lore.kernel.org/20250706193207.39810-1-sj@kernel.org

Signed-off-by: SeongJae Park <sj@kernel.org>
---
 hkml_cv_bogus/hkml_cv_bogus_hja33wk6 | 0
 1 file changed, 0 insertions(+), 0 deletions(-)
 create mode 100644 hkml_cv_bogus/hkml_cv_bogus_hja33wk6

diff --git a/hkml_cv_bogus/hkml_cv_bogus_hja33wk6 b/hkml_cv_bogus/hkml_cv_bogus_hja33wk6
new file mode 100644
index 000000000000..e69de29bb2d1
-- 
2.39.5

